doc/install.md

```diff
@@ -0,0 +1,265 @@
+   ---
+   apiVersion: rbac.authorization.k8s.io/v1
+   kind: ClusterRoleBinding
+   metadata:
+     name: scheduler-plugins-as-kube-scheduler
+   subjects:
+   - kind: ServiceAccount
+     name: scheduler-plugins
+     namespace: default
+   roleRef:
+     kind: ClusterRole
+     name: system:kube-scheduler
+     apiGroup: rbac.authorization.k8s.io
+   ---
+   apiVersion: rbac.authorization.k8s.io/v1
+   kind: ClusterRoleBinding
+   metadata:
+     name: scheduler-plugins-as-volume-scheduler
+   subjects:
+   - kind: ServiceAccount
+     name: scheduler-plugins
+     namespace: default
+   roleRef:
+     kind: ClusterRole
+     name: system:volume-scheduler
+     apiGroup: rbac.authorization.k8s.io
+   ---
+   apiVersion: v1
+   kind: ConfigMap
+   metadata:
+     name: coscheduling-config
+     namespace: default
+   data:
+     coscheduling-config.yaml: |
+       apiVersion: kubescheduler.config.k8s.io/v1alpha2
+       kind: KubeSchedulerConfiguration
+       leaderElection:
+         leaderElect: false
+       profiles:
+       - schedulerName: default-scheduler
+         plugins:
+           queueSort:
+             enabled:
+               - name: Coscheduling
+             disabled:
+               - name: "*"
+           preFilter:
+             enabled:
+               - name: Coscheduling
+           permit:
+             enabled:
+               - name: Coscheduling
+           unreserve:
+             enabled:
+               - name: Coscheduling
+   ---
+   apiVersion: apps/v1
+   kind: Deployment
+   metadata:
+     name: co-scheduling
+     namespace: default
+   spec:
+     selector:
+       matchLabels:
+         component: co-scheduling
+     replicas: 1
+     template:
+       metadata:
+         labels:
+           component: co-scheduling
+       spec:
+         serviceAccountName: scheduler-plugins
+         containers:
+         - args:
+           - --config=/etc/kubernetes/coscheduling-config.yaml
+           image: k8s.gcr.io/scheduler-plugins/kube-scheduler:v0.18.9
+           imagePullPolicy: IfNotPresent
+           name: co-scheduling
+           command: ['kube-scheduler']
+           resources:
+             requests:
+               cpu: '0.1'
+           securityContext:
+             privileged: false
+           volumeMounts:
+           - name: coscheduling-config
+             mountPath: /etc/kubernetes
+         hostNetwork: false
+         hostPID: false
+         volumes:
+         - name: coscheduling-config
+           configMap:
+             name: coscheduling-config
+   ```
+
+2. Check Coscheduling pod
+
+   ```bash
+   $ kubectl get pod -n default | grep co-scheduling
+   co-scheduling-95d4d54c6-986bl   1/1     Running      0          17m
+   ```
+
+### Stop Kubernetes default scheduler
+
+* Cluster created by `kubeadm` or `minikube`
+
+  1. Log on Master node and move `kube-scheduler.yaml` to upper directory.
+
+     ```bash
+     mv /etc/kubernetes/manifests/kube-scheduler.yaml /etc/kubernetes/
```

Comment:
Some concerns.
When the scheduler-plugins is deployed by deployment. And we stop the default-scheduler.
If the scheduler-plugins is abort because of some reasons like node not-ready or other things. Then the scheduler-plugins will be recreated by deployment. But no scheduler can schedule it.

Reply:
We may also replace the static pod by replacing kube-scheduler manifest

Reply:
@cwdsuzhou @denkensk Agree with you, I will update the doc about modifying /etc/kubernetes/manifests/kube-scheduler.yaml to avoid this problem.

Reply:
@cwdsuzhou @denkensk I wrote a new doc about how to modifying /etc/kubernetes/manifests/kube-scheduler.yaml. Could you help to review it？Thanks.